tests/Console/Report/FixReport/JunitReporterTest.php

```diff
@@ -175,10 +201,15 @@ class Foo
 
     protected static function createComplexReport(): string
     {
-        return <<<'XML'
+        $about = Application::getAppAbout();
+
+        return <<<XML
             <?xml version="1.0"?>
             <testsuites>
               <testsuite assertions="3" errors="0" failures="3" name="PHP CS Fixer" tests="2" time="1.234">
```

Comment:
Maybe we should use `Application::NAME` in place like this, instead of hardcoded `name` attribute?

Reply:
sometimes it's `PHP CS Fixer`, sometimes it's `PHP-CS-Fixer`. I think later is needed for some reporters which do not expect whitespace in given elements.

Let's keep as is and if someone wants to unify this or put behind variable, let's go with separated PR.